### PR TITLE
create overtime employee

### PR DESCRIPTION
**Pull request by thiennguyen96841996**

### CODE REVIEW

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

File: app/Http/Controllers/Employee/OvertimeController.php [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+         $overtimes = $this->overtimeRepository->getOvertimeUser();
```
Comment (BunnyPi04) [inline app/Http/Controllers/Employee/OvertimeController.php]: tab dư 1 space rồi

File: app/Http/Controllers/Employee/OvertimeController.php [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(OvertimeUpdateRequest $request, $id)
+    {
+         $date = $request->date;
```
Comment (BunnyPi04) [inline app/Http/Controllers/Employee/OvertimeController.php]: dư 1 space

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

**Pull request merged**